app.py

```diff
@@ -285,6 +285,7 @@ def main():
         protocol_main_chart_data = protocol_main_chart_data_mapping[protocol]
         if protocol_main_chart_data is None or protocol_main_chart_data.empty:
             logging.warning(f"No data for pair {debt_token} - {collateral_token} from {protocol}")
+            streamlit.subheader(f":warning: No liquidable debt for the {collateral_token} collateral token and the {debt_token} debt token exists on the lending protocol.")
             continue
         protocol_loans_data = protocol_loans_data_mapping[protocol]
         if main_chart_data.empty:
```

Comment:
Handle also the case when there is no data at all. For example, when the user selects only `Nostra Alpha` and the `STRK-USDC` pair.

```KeyError: 'Collateral'
Traceback:
File "/home/lukas/.cache/pypoetry/virtualenvs/derisk-research-Oy1qE-w4-py3.10/lib/python3.10/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 600, in _run_script
    exec(code, module.__dict__)
File "/home/lukas/derisk-research/app.py", line 520, in <module>
    main()
File "/home/lukas/derisk-research/app.py", line 305, in main
    loans_data['Collateral'] = loans_data['Collateral'].apply(parse_token_amounts)
File "/home/lukas/.cache/pypoetry/virtualenvs/derisk-research-Oy1qE-w4-py3.10/lib/python3.10/site-packages/pandas/core/frame.py", line 4102, in __getitem__
    indexer = self.columns.get_loc(key)
File "/home/lukas/.cache/pypoetry/virtualenvs/derisk-research-Oy1qE-w4-py3.10/lib/python3.10/site-packages/pandas/core/indexes/range.py", line 417, in get_loc
    raise KeyError(key)```